lambda-multi-svc/README.md

```diff
@@ -189,15 +197,85 @@ aws proton create-environment \
   --spec file://specs/env-spec.yaml
 ```
 
-Wait for the environment to successfully deploy. Use this call to check for deployment status:
+Wait for the environment to successfully deploy. Use the `get` call to check for deployment status:
 
 ```bash
+aws proton wait environment-deployed \
+  --region us-west-2 \
+  --name "multi-svc-beta"
+  
 aws proton get-environment \
   --region us-west-2 \
   --name "multi-svc-beta"
 ```
 
-Then, create a Proton service and deploy it into your Proton environment.  This command reads your service spec at `specs/svc-spec.yaml`, merges it with the service template created above, and deploys the resources in CloudFormation stacks in your AWS account using the Proton service role.  The service will provision a Lambda-based CRUD API endpoint and a CodePipeline pipeline to deploy your application code.
+### Create Environment in one account and Provision in another account
+
+First, log into the environment account where you want to provision the environment resources and create the IAM role that Proton will assume to provision resources and manage AWS CloudFormation stacks in your AWS account.
+This can also be done from the console. https://docs.aws.amazon.com/proton/latest/adminguide/security_iam_service-role-policy-examples.html#proton-svc-role
+
+```bash
+aws iam create-role \
+  --role-name ProtonServiceRole \
+  --assume-role-policy-document file://./policies/proton-service-assume-policy.json
+
+aws iam attach-role-policy \
+  --role-name ProtonServiceRole \
+  --policy-arn arn:aws:iam::aws:policy/AdministratorAccess
+```
+
+Then, create and send an environment account connection request to your management account. When the request is accepted, AWS Proton can use the associated IAM role that permits environment resource provisioning in the associated environment account. 
+You need to specify the environment name that you will use for the environment.
+
+```bash
+aws proton create-environment-account-connection \
+  --region us-west-2 \
+  --management-account-id ${management_account_id} \
+  --environment-name "multi-svc-beta" \
+  --role-arn arn:aws:iam::${environment_account_id}:role/ProtonServiceRole
+```
+
+Log into the management account and accept the environment account connection request from your environment account. This can also be done from the console. 
```

Comment:
do we want to include the instructions here? as in ada command sample or copying isengard credentials etc

Reply:
I think both ada and isengard are our internal tools for user credentials. Customers should know how to log into their accounts and maybe they have different solutions for temporary credentials.